master/docs/developer/www.rst

```diff
@@ -620,7 +620,7 @@ Several methods are added to each "restangularized" objects, aside from get(), p
 DataService
 .............
 
-DataService is the future replacement of BuildbotService for accessing the Buildbot data API.
+DataService is the replacement of BuildbotService for accessing the Buildbot data API.
 It has a modern interface for accessing data. It uses IndexedDB for storing cached data as a single data store,
```

Comment:
While you are at it, could you please bring it up to the one sentence per line policy?